katello-selinux-relabel

```diff
@@ -1,26 +1,6 @@
 #!/bin/bash
 
-# relabel foreman
```

Comment:
Should it call `foreman-selinux-relabel`?

Reply:
Hmmm you are right, let me brainstorm here. As a user, you want to have everything to be relabeled if you do:

    # foreman-selinux-relabel

Therefore we should perhaps do the opposite - to call this script from foreman's one. Now, we don't want to create dependency like that, we should perhaps go with either some relabel.d/ directory mechanism, or simply do call all scripts which match this pattern: /usr/sbin/*-selinux-relabel. Now that's a bad idea, I think we need to come up with relabel.d/ or similar approach.

Alternatively, we can leave this as a dummy script that will fail with "Please use foreman-selinux-relabel" and move the line there. Opinions?

Reply:
I am for the easier option, @ekohl what do you suggest?

Reply:
Calling scripts from .d directory is quite easy... It's just packaging work there: https://github.com/theforeman/foreman/blob/develop/script/foreman-debug#L375-L385